tools/dependency/release_dates.py

```diff
@@ -86,12 +86,18 @@ def create_issues(dep, metadata_version, release_date, latest_release):
             labels.append(label.name)
     if len(labels) != len(LABELS):
         raise DependencyUpdateError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
-    body = f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>:{latest_release.tag_name}@<{latest_release.created_at}>'
-    title = f'Newer release available {dep}: {latest_release.tag_name}'
+    body = f'''\
```

Comment:
the indent here is causing the comment to be quoted, which i think we dont really want, and means the url doesnt get rendered as a url

Reply:
i would suggest moving the comment into a template

you could do this with a `string.Template` altho we are increasingly using `jinja2`, so that would work too (you might need to add a dep in requirements.txt in that case)

Reply:
let me try this with `string.Template`  if that works i dont have to deal with external dependency and its management then.

Reply:
let me try this with `string.Template`  if that works i dont have to deal with external dependency and its management then.

Reply:
so i tried things with `string.Template`  didn't work with it either. 
dedent is the fast solution for this ig. 

https://github.com/ME-ON1/envoy/issues/71

Made this issue with dedent  towards right side and link was working